Android/ChorusRFLaptimer/app/src/main/java/app/andrey_voroshkov/chorus_laptimer/PilotsRssiListAdapter.java

```diff
@@ -133,10 +133,11 @@ public void onCheckedChanged(CompoundButton buttonView, boolean isChecked) {
         DeviceState ds = AppState.getInstance().deviceStates.get(position);
         String ch = AppState.getInstance().getChannelText(position);
         String band = AppState.getInstance().getBandText(position);
+        String freq = AppState.getInstance().getFrequencyText(position);
         Boolean isEnabled = AppState.getInstance().getIsPilotEnabled(position);
 
         viewHolder.txtThresh.setText(Integer.toString(ds.threshold));
-        viewHolder.txtChannelLabel.setText("Channel: #" + ch + " (" + band + ")");
+        viewHolder.txtChannelLabel.setText("Channel: #" + ch + " " + freq +" (" + band + " band)");

```

Comment:
Let's put frequency after the band, please (I think it will be a bit more readable)